### ISSUE TITLE
Script for GIFs/ Best trajecotrie / loss function

### ISSUE THREAD

**Issue by spam-12345** [HarshayuGirase/PECNet#41]
Thank you for your wonderful code and paper! 
Would it be possible that you also provide the script for generating the gifs for the YNet?
In addition, I am unfortunately not quite clear how to find out which is the best trajectory of the predicted 20 (also for the YNet)?

**Comment by ArcaneEmergence** [comment]
For Y-net at least the GIF was done with matplotlib for each single timestep and external web tools (you can just google GIF maker) to create the GIF from the single images

**Comment by spam-12345** [comment]
Generally creating a GIF is not the problem. However, on my dataset I don't get as nice heatmap representations as you do. They don't have such strong separated color dots (see picture). It's more of an even gradient. That's why I had asked.

![image](https://user-images.githubusercontent.com/48084588/158977433-1618b974-fadc-4b8b-ad3c-bfcec5d4864c.png)

The more important question is the second one. How do I know which is the "best" trajectory? In the test.py script you evaluate the first trajectory with the softargmax on softmax map function (line 165 and following). Is this the crucial clue?

And then I have another question. In your paper you write that you use the trajectory, the goal and waypoints for the loss (with different weights lambda). In the script train.py (around line 88), I can't see the different weights for goal and waypoints/trajectory..

Thanks for your help!

**Comment by HarshadZade** [comment]
Hi [spam-12345](https://github.com/spam-12345), can you please tell me how you used your own data? Like, how did you preprocess it? Thank you.
